src/bytes.rs

```diff
@@ -545,7 +613,7 @@ impl Bytes {
     /// assert_eq!(bytes.try_into_mut(), Ok(BytesMut::from(&b"hello"[..])));
     /// ```
     pub fn try_into_mut(self) -> Result<BytesMut, Bytes> {
-        if self.is_unique() {
+        if unsafe { (self.vtable.cheap_into_mut)(&self.data) } {
```

Comment:
Should we change `is_unique` to always return `false` instead of adding this extra check? That's how `from_static` works.

Reply:
Sounds sensible, let me know and I can remove `cheap_into_mut`.

Reply:
done